src/components/Accounts/ViewFeesFines/ViewFeesFines.js

```diff
@@ -209,51 +211,42 @@ class ViewFeesFines extends React.Component {
 
     const allChecked = _.size(checkedAccounts) === this.props.accounts.length;
     this.setState({ allChecked });
-    const values = Object.values(checkedAccounts);
-    let selected = 0;
-    values.forEach((v) => {
-      selected += (v.remaining * 100);
-    });
-
-    selected /= 100;
-    this.props.onChangeSelected(parseFloat(selected).toFixed(2), values);
-
-    const open = selected > 0;
-    const closed = values.length > 0;
-    this.props.onChangeActions({
-      waive: open,
-      transfer: open,
-      refund: open || closed,
-      regularpayment: open,
-    });
+    this.updateToggle(checkedAccounts);
   }
 
   toggleAll(e) {
     const accounts = this.props.accounts;
     const checkedAccounts = (e.target.checked)
       ? accounts.reduce((memo, a) => (Object.assign(memo, { [a.id]: a })), {})
       : {};
+
+    this.updateToggle(checkedAccounts);
+    this.setState(({ allChecked }) => ({
+      allChecked: !allChecked
+    }));
+  }
+
+  updateToggle(checkedAccounts) {
```

Comment:
It is difficult to guess what the `updateToggle` function will do. Typically, functions named like `toggleFoo()` will flip the boolean value they are associated with, but `updateToggle` doesn't quite fit that naming-pattern, and isn't as simple as flipping a boolean value.